README.md

```diff
@@ -464,7 +470,7 @@ class MyEstimator:
 becomes:
 
 ``` py
-from sklearn_compat.base import ParamsValidationMixin
+from sklearn_compat.base import ParamsValidationMixin, _fit_context
 
 class MyEstimator(ParamsValidationMixin):
```

Comment:
this should also inherit from `BaseEstimator`, and that interacts with `ParamsValidationMixin`